src/test/java/mekanism/api/math/FloatingLongTest.java

```diff
@@ -78,6 +78,14 @@ void testDivisionLargeNumerator() {
         Assertions.assertEquals(FloatingLong.create((long) 649_657 * 337), a.divide(b));
     }
 
+    @Test
+    @DisplayName("Test division with a very large denominator")
+    void testDivisionLargeDenominator() {
+        FloatingLong a = FloatingLong.create(922355340224119L);
+        FloatingLong b = FloatingLong.create(-1L);
+        Assertions.assertEquals(FloatingLong.create(0L, (short)1), a.divide(b));
```

Comment:
Denominator is the part under a fraction (a.k.a. divisor) but it's being used as the Numerator in this division. I might be missing something here though?

Reply:
FloatingLong uses unsigned longs internally so -1 is largest whole number FloatingLong

Reply:
Ah I see, sorry for wasting time.

Reply:
Sorry not sure I understand, is it the -1L that is confusing? For this test to give useful information I had to also include a decently large numerator (>= (2^64-1) / 10,000)?, otherwise the result will just be zero due to the minimum nonzero value of 0.0001. 
It's really the -1L (maximum unsigned long value) that I wanted to test. Let me know if I'm understanding your question incorrectly :)

Reply:
I misinteperated `a` as being the larger number.

Reply:
No worries! Happy to help/explain